### ISSUE TITLE
[COMMUNITY DISCUSSION] Should @HNS twitter be community-operated for marketing purposes or remain inactive indefinitely?

### ISSUE THREAD

**Issue by Chjango** [handshake-org/handshake-web#14]
Opening up this topic for an informal discussion since there's been ephemeral conversations about this happening over Telegram.

In my personal view, I view Twitter as an effective marketing tool to address a crypto audience. Crypto Twitter is where most new retail investors look to to find information they can't get elsewhere. Keeping the @HNS handle inactive indefinitely is leaving the opportunity to speak to that audience on the table.

From a decentralization perspective, the factor that matters most is whether the protocol—and the governance over that protocol—is sufficiently decentralized. Whether or not a Twitter handle is communicating updates about said protocol ranks much lower on the list of factors.

There were concerns raised [[1](https://twitter.com/dwebASAP/status/1388759892127793155?s=19)] [[2](https://twitter.com/dwebASAP/status/1388647979523792899?s=19)] about the inactivity of @HNS being perceived—to the detriment of Handshake—as Handshake being a "dead project" or a "ghost chain". You and I know that this certainly isn't the case; but this is the reality of the optics.

From a market standpoint, there are 99,999 other projects vying for retail attention. And retail pays attention to Twitter. If we care about Handshake price, then it should be competing for some of that attention. If it remains inactive indefinitely, then, at least for the short run, Handshake runs the risk of not being discovered while we're in the heat of a DeFi and dWeb summer.

That's my vote. Would love to hear your input, and looking forward to having a constructive discussion. The fact that we're having this discussion merits a healthy ecosystem already.

Onward!

**Comment by pinheadmz** [comment]
Thanks so much for opening this discussion, soliciting input from the community, and being a friendly benevolent custodian of a very important asset - with over 14,000 followers. My only input to the @HNS policy is: please tweet hsd and hnsd releases, and especially critical security update announcements like the recent [inflation bug patch.](https://handshake.org/notice/2020-04-02-Inflation-Bug-Disclosure.html)

**Comment by johnnywu-namebase** [comment]
I'm all for the @HNS Twitter account being more active, the main thing is who would manage it? The dWeb Foundation? 

It's a super big role to play and any position the @HNS Twitter account takes regardless of how minor will be seen by outsiders as representing the entire Handshake community's position. For example, I don't think the account should "like" a Tweet that even suggests a roasts towards another project, which makes it a tricky decision on whether to promote community events like #monthofmemes. 

That said, I think @HNS should help promote objectively positive community events such as HandyCon or the Flamingo Handshake auctions.

**Comment by smcki012** [comment]
The greatest net positive returns for the Handshake community (especially in regards to social graphs and reach), would be if keystone community, technical, business, and educational stakeholders amass and refine their own social reach and referent authority/social legitimacy by directly engaging and interfacing with the community at large.

Great examples of this would be Kiba Gateaux, Mark Smith, and Handshake Jesus: all have built a loyal community following and social legitimacy based on their perceived inputs and how they interfaced with it using their accrued social capital.

If you wish for Handshake to stand on its own merits, and not create socio-technical debt, the best course of action would be to leave the handle vacant, and just put a bot to automatically share all latest releases/merges/and emergency patches from the hsd reference implementation. However, even this is dangerous, as it creates social authority around 1 fullnode client, and puts it on a social pedestal above alternate implementations -- something that has irked JJ himself for a long time in Bitcoin, from my recollection.

14,000 people is nothing without impressions, and without a consistent ethos and messaging, that account is mostly just a glorified sounding board for technical releases. There would be better net returns socially if the community capitalized on bringing in younger and more adventurous/boisterous types to echo the communities' messaging and hardwork, and celebrating those who have done a great job of it thus far.

A realignment of where we place our values and appreciation for those who have already created a lot of traction would do us much better than coming to a contentious social decision that just ends up an opportunity for traders to profit off the angst and uncertainty; Handshake fixed DNS, not human psychology.

**Comment by DIPMR** [comment]
Best would be to take the link off the official website and use it, I would say even take off the Reddit link too. Link only the Github, that's Handshake and that's what world needs.

**Comment by alchmind** [comment]
I agree with the opinion that the project should shine on its own merits, but people have invested in this coin not only for the strong tech but also for the chain design. The more people we have investing in the coin, the more the project can self-perpetuate with the airdrop funding mechanism and bring in even more developers (and make it worth their time). I don't like speculative fervor in the crypto markets (if that is a concern among community members), but I'd argue we're not going to spark rampant speculation just by posting about our developments. We're not going to get everyone to understand the tech and its philosophy the way the core community does, but that's not our main concern. Our main concern is making sure that we achieve the goals of decentralization, censorship-resistance, and the free exchange of information to be had, and that people can experience that. We can at least use the account to post TheShake weekly or something so people know the progress. People just need to know we're not dead, and being that HNS tech is running in the background most of the time, no one will notice unless we broadcast this.

We can have the greatest tech in the world but if no one knows about it, no one will care. Right now, our main concern should be reaching a critical mass of adoption and legacy players to adopt after that. We're not doing coin holders or the dev community any favors by holding back the @HNS account.

**Comment by troq** [comment]
Handshake not having a centralized organization driving it carries some weaknesses but also enables Handshake's greatest strength — a resilient decentralized community that makes Handshake one of the strongest communities in crypto. @HNS changing course to become active threatens that strength because it's a centralized point of authority in the community and introduces governance concerns. Handshake is already seeing great progress both in community growth (HandyCon was a huge success) and industry adoption (NameCheap is now purchasing Handshake domains and hodling HNS). Instead of introducing a point of centralization into a decentralized community, we should solve the root problem you mentioned of people looking at @HNS and thinking Handshake is inactive; I recommend that whoever controls @HNS pin a post that says that there is no official twitter account for Handshake because it is a thriving decentralized community.

**Comment by peepohns** [comment]
I agree with Chjango.

**Comment by agaamin** [comment]
I think the twitter handle should be used, and used heavily to promote Handshake.

IMHO the best thing about HNS is not the decentralization and and standard crypto benefits list that gets touted, instead It's the immense potential it brings in terms of possibilities to build new tools, products and services. If handshake needs to be useful to humanity it needs to be mass and mainstream.  And it does not help the cause to for some reason to avoid one of the most popular platforms of crypto verse. 

Also The idea that Just because someone manages a social media handle it would "centralize" Handshake is simply not true. I manage the Facebook, Insta and Linked in Profiles for Handshake. In fact recently i have ensured its presence on Hive and Steem blog ecosystem also with Handshake community. I don't see how these handles are going to take a decentralized product like Handshake and make it centralized. 

In fact we should use as many tools that are out there to create awareness about Handshake. And twitter is one of the important avenues. If it helps we could perhaps create a community that manages these social handles so that fears of centralization ( if any) can be addressed.

**Comment by Anunayj** [comment]
Personally imo the @hns twitter handle should only be used for announcing Blockchain related changes (Bugs, soft forks, potential future hard forks). Any marketing related announcements can be done from unofficial individual controlled accounts, (say @HNSUnofficial ,which can be given a shoutout from the @hns handle).

The reason behind my opinion is, having a official twitter handle makes handshake feel not so different from ICANN/other centralized naming systems. It gives the view that the person operating the twitter handle somehow speaks for the community.

In the short I can see how marketing the coin can help with the popularity of the coin, and increase in it's price. But in the long term, I do not think the goal of Handshake is to be *yet* another cryptocurrency going to the Moon, but be a alternative decentralized naming system that doesn't just favor the corporations.

**Comment by Falci** [comment]
My opinion: no use is also a message. But it's a negative message.

**Comment by evbots** [comment]
Shifting operations of the account away from "the founders" and towards a for-profit company or a nonprofit org with transparent membership would do a lot to solve the problem of the public thinking that account "speaks for the community." At least then people would be able to judge the intent behind the tweets a little easier. Regardless of the path - it should be very clear who is behind it and the current "status." the situation now is just confusing.

**Comment by metanoia64** [comment]
Over the last year HNS twitter handle following increased 131%. There is interest. Source: https://www.coingecko.com/en/coins/handshake#social

**Comment by tynes** [comment]
> In my personal view, I view Twitter as an effective marketing tool to address a crypto audience. Crypto Twitter is where most new retail investors look to to find information they can't get elsewhere. Keeping the @hns handle inactive indefinitely is leaving the opportunity to speak to that audience on the table.

I agree with this generally but what sorts of community members do we want? Do retail investors add value to the community as a whole or are they there to attempt to extract value? (and then get dumped on by insiders)

> From a decentralization perspective, the factor that matters most is whether the protocol—and the governance over that protocol—is sufficiently decentralized. Whether or not a Twitter handle is communicating updates about said protocol ranks much lower on the list of factors.

The perception that the community holds directly influences the governance. Communicating updates about the protocol is not speaking to retail investors. Those are very different modes of communication. Is there an expectation that retail investors will not participate in the governance in the future? Will their initial perceptions of the community not impact the way that they (the small % that do become active community members) try to govern and participate in the community?

> There were concerns raised [1] [2] about the inactivity of @hns being perceived—to the detriment of Handshake—as Handshake being a "dead project" or a "ghost chain". You and I know that this certainly isn't the case; but this is the reality of the optics.

This is disappointing but there needs to be a killer app for Handshake that just isn't possible anywhere else. Right now the main usecase of Handshake seems to be domain speculation which isn't particularly adding value to the world. The potential usecase of making domain names into cryptographic assets is huge. Getting to this point is going to take some time.

In my opinion, Handshake isn't a project for retail investors to yolo buy without doing research. Its the kind of project that developers need to adopt first as part of the tech stack and natural utility will lead to price appreciation. I doubt that retail investors would add to the community (unless you count making the insiders rich) and Handshake will never outcompete the flavor of the month defi farm that spends all of its time attempting to sell to retail.

Making the insiders rich does bring funding generally into the Handshake ecosystem but there is no guarantee that the money will be re-invested. Optimizing for marketing towards retail investors puts a lot of faith that money will be reinvested into the ecosystem at the expense of having a credibly neutral, long term reputation of the project.

I generally agree with @troq and think that its more important to think about the long term. Would namecheap have joined the community if the twitter handler was tweeting to retail? What future community participants wouldn't join if the twitter handle started doing this? Being credibly neutral about "pumping price" from legitimate sources is important for a project to be taken seriously by the people that matter. The infrastructure people will be turned off by tweets that appeal to retail.

One interesting point is that Flashbots doesn't have a twitter and its working fine for them. Its a lot of responsibility to do a good job at the voice of authority. Existing Handshake influencers could work together to get views on some random twitter account that they make.

> The fact that we're having this discussion merits a healthy ecosystem already.

You can never just be in a state of healthy ecosystem. Its a constant battle to maintain a healthy ecosystem - continual effort over time.

**Comment by pinheadmz** [comment]
I agree with @tynes as well. I would like to see a heartbeat from @HNS but I think it should have no personality and generally be boring. The personality comes from the sparkling creative community! As @smcki012 mentioned, several influencers have already emerged and do great work to promote HNS without any kind of "official" stamp.

To expand on @tynes comments about the retail audience, I think @HNS should be forbidden from low-effort comments about price. [This tweet](https://twitter.com/HNS/status/1390041107422056450?s=20) is well-intentioned but tagged with "#hodlhands" which in my opinion is inappropriate and unprofessional. "HODL" means "buy this without thinking, never use it and never sell it." That sentiment seems antithetical to the everything this project is designed for, disregards all the amazing work by contributors from all over the community building exciting use cases for Handshake, and puts HNS in a box along with every other blockchain project that adds zero features to the ecosystem. 

It seems like there is a general feeling that @HNS should still do *something*, and so the original question is approaching an answer. I think the next step is a community drafted and approved policy that the @HNS custodian must follow.

We can leave this issue open for a while to collect more comments, then the next thing to do I think is:

@Chjango opens a pull request to this repo that adds a new html page (i.e. `handshake.org/twitter.html`) that specifies the behavior of @HNS with little room for interpretation. This will make the custodian's job easy, and provide continued influence by the community (who can open new PRs to update that policy).

Examples:

- No hashtags or cashtags
- No price commentary, speculation or "buy and hold" comments
- No competitive comments targeted at other blockchain naming systems 
- Retweets:
  - Never with additional comment
  - Software update announcements from handshake-org, kyokan, handyminer, sinpapeles, hmail, namebase and any other service that clearly supports more than (lets say) 100 handshake users
  - Influencers in the community that clearly provide financing or education

**Comment by FistFullOfAss** [comment]
Agree with @pinheadmz and @tynes the twitter shouldn't focus on retail attention, but developer attention instead. Along with community events like Handycon and Flamingo handshake and updates regarding hsd and hnsd, I think the Twitter should also be reminding devs that the airdrop is there for them to claim. The airdrop is Handshake's main distribution mechanism that will incentives new devs in the community and hopefully more websites on Handshake domains and products working with handshake. The promotion of products like Hmail, hsnsearch, and XNHNS would also be fine in my eyes to show the already growning ecosystem around Handshake names. Trancparancy is also key, whoever the admins of the account are should be easy find and to communicate with and should be able to be replaced through some type of community consensus.

**Comment by hns-tld** [comment]
I agree with @agaamin that we should use as many tools as possible but we should communicate only 1 concise message.

@smcki012 suggestion of a bot make sense and comes with its own set of issues but at least it will not be left up to individuals.

This will deter messaging that could be bias or have an agenda that could be harmful to HNS and all those invested in its success.

Remember:

"Every single person is a genuine and authorized Director of The Handshake Project. If someone has given you a document or business card representing themselves as a Director of The Handshake Project, they have full authority to represent as Director of The Handshake Project with their own personal viewpoint, So Please Treat Them Right. You, the reader, are also a Director of The Handshake Project and have equal claim on authority, action, and viewpoints."

@HNS should detail the how to, technical, technology, projects and anything else that is not related to individuals.  

I suggest that as a community we are all responsible to spread branding, tech news, memes and overall marketing of HNS.

**Comment by brandondees** [comment]
I agree with the rationale behind keeping the account dead/burned, but I believe it needs to update profile / pinned message to explain that situation very clearly. The handshake.org site should probably have a page dedicated to explaining in more detail that it can link to, where we can more easily update and refine the messaging, including to suggest checking out the various non-authoritative alternative channels of engagement that will evolve over time. Perhaps older tweets should be cleaned up as well to reduce confusing more people. Links to it as the official handle should be removed from wherever they've proliferated such as coingecko et al. Orgs like handshake institute and dweb foundation can use their own independent accounts to fulfil its expected role of community announcements and marketing engagement, with less confusion and conflict.

**Comment by kibagateaux** [comment]
Agree with @tynes on "who are we targeting". Optimizing for 0 IQ retail is a race to the bottom with every other shitcoin on crypto twitter. Encirca and namecheap are very involved with Handshake now and they did not need a twitter account to see the value of what was created or the community that has grown. If we can convince multi-billion dollar companies without a twitter account we don't need one.

As @smcki012 mentioned, if there was an official Twitter account the community would not be as strong s it is today. I would not have gathered the early following that I did and without becoming a twitter thot leader I would not have taken the initiative to lead the community outside of twitter as well. I would have assumed that someone running @hns also meant someone was running ecosystem development and I never would have started HNS Fund which is currently the largest source of funding for the HNS community. 

Handshake is built different. Just like we are not a simple copy paste of Bitcoin we can not copy paste simplistic marketing tactics and strategies.

Whoever is running @hns currently *cough cough* is not credibly neutral which is a necessary quality of any blockchain system, community included. Primary example is they have retweeted dweb foundation but not HNS Fund even though by all objective, quantifiable metrics HNS Fund has more community engagement with more HNS donated (~75k HNS + 25 ETH vs ~10k HNS ) and more twitter followers (257 vs 243). This shows the owner of @hns is trying to push their own objectives and narrative that does not align with the ideals of the community.

They're running a standard psyops campaign, this thread is part of it. They just recently started posting again to give community a taste of what could be and are now trying to get consent to have free reign to post whatever they want which will lead to spreading their own propaganda to influence the direction of Handshake. This is unacceptable and should not be tolerated.

If someone was supposed to speak on behalf of the Handshake project then the founders would have created a centralized foundation. They didn't. @HNS twitter us dead and the Handshake project lives on more vibrantly because of it.

Pin a tweet saying the account is not active and link to $HNS and #HNS (no specific accounts) for the latest news and leave it at that.

**Comment by NetOpWibby** [comment]
Retail is fickle and prone to massive swings in emotion. I don't think we should cater to the lowest common denominator.

Like others have said, make the account just tweet out code updates. Otherwise, I agree with Kiba.

Anectdotally, I don't see people tagging the official Handshake account. Rather, they (and I) just use `#handshake` and `$hns`.

**Comment by taylormartens** [comment]
Until Handshake sees mass adoption, not using the @hns Twitter to promote and document handshake related projects will be a net negative. 

I believe there will be an inflection point where people on boarding to handshake are not native to crypto. At that point having an official Twitter will be a great resource to point new users to the various things in the ecosystem.

**Comment by namervin** [comment]
💯% embrace it at least monthly if not weekly. Just a bad first vibe that can create a bounce if that’s what they see from a tag or search if the project looks like a ghost town.

**Comment by tiMaxal** [comment]
While only having started becoming part of this community a month or so ago, one of the things that appealed was the brevity and apparent non-politicisation of 'official' social media; keep it simple and keep it true, and if it _is_ going to kept going moving forward, keep it active within the month if not the week. Perspective of a recent 'outsider' :-).

**Comment by FistFullOfAss** [comment]
starting to lean more with @kibagateaux and @smcki012 on this. To leave the @hns twitter in the hands of a few individuals with no proper governance tools to decide this is near impossible. If this task is near impossible I say we clean up the Twitter, Make a pinned tweet explaining the situation (and/or a page as @brandondees suggested), then kill the twitter and let the community advertise and grow through its grass roots. New people that make that extra effort to learn will be guided by the community to all the resources they may need, the people to follow for all things Handshake, and to stay and build. Those that are turned off by this fact just want numba to go up fast. This community is already so vibrant without the @hns twitter, staying dead won't stop the community from becoming bigger and better. I say the next steps for us should be drafting the message that will explain why the twitter is the way it is.

**Comment by DIPMR** [comment]
So who controls Twitter and Reddit handles?

**Comment by pinheadmz** [comment]
> So who controls Twitter and Reddit handles?

@DIPMR 

[The founders](https://old.reddit.com/r/handshake/comments/n9t5ga/who_is_the_founders_of_handshake/) set up the IRC channel, the subreddit and twitter handle long before mainnet launch. I'm a mod of #handshake and /r/handshake along with them. Access to the twitter handle was passed around for a while and @Chjango (who opened this issue for discussion) either has access to it now, or access to whoever has access to it, delegated by the founders.

**Comment by DIPMR** [comment]
And when and where did this transfer/access to Chjango happen? Why can't anybody else has the access?

**Comment by Falci** [comment]
Another opinion: someone has the domain `hns` and it isn't "the Handshake", it belongs to someone in the audience. That doesn't make it "the official Handshake domain".

Same could be true for Twiiter.

handshake.org should not link nor endorse the `@hns` on Twitter and whoever controls it has the rights to publish his/her opinion as a Handshake director. See it as a perk for the early adoption.

It should be the rule for all external resources: handshake.org should not link to block explorers, wallets, exchanges nor communities (Telegram, Discord, Reddit or whatever). A link means endorsement.

---
Another way to see this comment: imagine that the founders didn’t get `@hns` on twitter in the early days, and that eventually somebody did. Would we have this discussion now? I don't think so.

**Comment by evbots** [comment]
> Another opinion: someone has the domain `hns` and it isn't "the Handshake", it belongs to someone in the audience. That doesn't make it "the official Handshake domain".
> 
> Same could be true for Twiiter.
> 
> handshake.org should not link nor endorse the `@hns` on Twitter and whoever controls it has the rights to publish his/her opinion as a Handshake director. See it as a perk for the early adoption.
> 
> It should be the rule for all external resources: handshake.org should not link to block explorers, wallets, exchanges nor communities (Telegram, Discord, Reddit or whatever). A link means endorsement.
> 
> Another way to see this comment: imagine that the founders didn’t get `@hns` on twitter in the early days, and that eventually somebody did. Would we have this discussion now? I don't think so.

agreed. take it away from the founders and you remove the inclination to view the "@hns" twitter handle as official in any way. but ideally it would still exist, as "HNS" is now the most commonly used ticker symbol - it's valuable and could be put to good use. the implicit desire to view this as the project's official account is the problem here.

**Comment by DIPMR** [comment]
I agree with @Falci

**Comment by johnnywu-namebase** [comment]
I think a pinned Tweet on the @HNS Twitter account sharing that the account is intentionally deactivated so the visitor should reference $HNS instead is best for the Twitter account.

> It should be the rule for all external resources: handshake.org should not link to block explorers, wallets, exchanges nor communities (Telegram, Discord, Reddit or whatever). A link means endorsement.

I don't know if I agree with this because it'd make discovering community channels from the handshake.org website (which I believe most people first visit) really difficult. Maybe we can do something like other projects do where we just make it very explicit that they're all community-managed channels and link out to even more resources than there current are (e.g. blockexplorer.com, gateway.io, namebase.io, etc.). Maybe anyone can make a PR to add a link to their project and it can just be ranked by number of visits? Just a thought. 

If we do go ahead with removing all links, then the pinned Tweet can also link to handshake.org/community which will explain the reason behind its inactivity.

**Comment by pinheadmz** [comment]
handshake.org is already community-managed, just like hsd itself. The proposal to add Bob Wallet (which is really hsd with a GUI paid for by the founders) had 5 comments by as many participants. Adding telegram had 2 participants and of course the word "experimental" on the front page brought in comments from more than 7 people. Removing links from handshake.org is an option, but it should be submitted as a pull request and debated by the community. This is like local politics to me -- _kiiiinda_ wish more people were involved, but it is not actually a dictatorship.

Back to the twitter handle -- I think killing it with a gravestone is not a bad idea... but I seriously believe the community can do better. We are a COMMUNITY, we work together to be something big. Everyone on this thread has experience in teamwork, DAOs, etc - the twitter handle *can* be used to benefit everybody and I believe the 22 directors on this thread so far can figure out how to do it.

**Comment by DIPMR** [comment]
Either gravestone it  or disable links.Who knows who gets the power next and does what with it. So many questions...I still didn't get answer to my question about handing access to Chjango? 


Either it should not be active or everybody should have equal right on it. If somebody can get access to it then What logic   stops me from accessing the handle.

**Comment by FistFullOfAss** [comment]
Even if the Twitter rebranded as community run or unofficial because of the current situation and the amount of followers it has, it will still have more social legitimacy than any one person/couple of people in the community. Who and what is going to decide who runs the twitter without any governance tools? who is to decide that an incumbent has more say in how Handshake is perceived than a newcomer?

As a community we can already share what we think is important through likes and retweets on Twitter, giving social legitemancy to people in the community in a better way. The fact that we are talking like it has this power to influence is more than enough reason to kill it.

**Comment by chrxn** [comment]
> From a decentralization perspective, the factor that matters most is whether the protocol—and the governance over that protocol—is sufficiently decentralized. Whether or not a Twitter handle is communicating updates about said protocol ranks much lower on the list of factors. -- @Chjango

I have to disagree with @Chjango on this point -- if this `@hns` twitter handle is perceived as the "official" account, then it acts as a funnel into the ecosystem and routes attention. There is a lot of leverage to be wielded there for whomever controls it, and the way that self-interested items can injected there can be subtle. 

If we are committed to decentralization in the protocol and governance, then we shouldn't delude ourselves into thinking that  centralized control of an official source of messaging is anything but a compromise of those values and a vector of capture for self-interest.

The truth of the matter is that there are many voices advocating for (different) future(s) of Handshake -- anyone that is not aware of that should take note. Handshake 'succeeding' or gaining widespread popularity can take many forms, and how we engage this particular type of issue under discussion here is a strong determining factor of that.

Each of these voices say that they 'want Handshake to win' or similar. However, each has a different set of values and motives that color their actions -- **and that isn't a bad thing**, it just is the reality of the situation. 

- dWeb Foundation
- Namebase
- HNS Fund
- Handshake Institute
- etc.

When you hand over the keys of something like `@hns` over to one person or organization you are putting their values over other voices in the ecosystem and asking the rest of the community to trust them.  You let them decide where to route the attention. 

I think that would be a tragic mistake. There's a better way that doesn't require us to compromise on decentralization and supporting diverse voices in the community with integrity.

> The greatest net positive returns for the Handshake community (especially in regards to social graphs and reach), would be if keystone community, technical, business, and educational stakeholders amass and refine their own social reach and referent authority/social legitimacy by directly engaging and interfacing with the community at large. -- @smcki012

I agree completely with @smcki012

I think that the next step of maturation for our community is to understand ourselves in terms of what distinct values and missions are being put forward and getting involved with them to promote Handshake in the way that each individual wants to amplify by adding support to that independent take on how to grow Handshake.

The [dWeb Foundation](http://decentralizedinter.net) is doing valuable work to build out means of access for retail into the ecosystem and to foster new projects in HNS, and that is important and appreciated. (my own perception of their focus) If you personally are excited about that and resonate with them, then you would do well to contribute to their efforts, retweet them and help them gain influence on Twitter so that they can increase the surface area and efficacy of their actions along the lines of their mission and values. 

The [Handshake Institute](https://handshake.institute) (full disclosure: I founded the Institute with HandshakeJesus) and [HNS Fund](https://hnsfund.titansofdata.org) are focused on building resilience and decentralized products and services in cooperation with community members and creators that are already engaged in work. Creating educational material and outreach for Handshake is one of [our values](https://handshake.institute/about/#values) and a part of our mission as an educational non-profit. If you resonate with our values and want to help bring people into Handshake via decentralized products & open source focused education, then maybe consider joining us and supporting our work by getting in touch. Build something and collaborate with us.

I think the best way forward is to support the voices you resonate with and amplify them.

---

With respect to the @HNS twitter handle -- for simplicity sake, I would accept it being made into a gravestone. I think that's the easiest route.

## alternative?
A much more effort intensive alternative that could perhaps be implemented over the long term --

We could create a bot repo that tweeted messages aligned with a neutral, non-partisan educational campaign that had a broad coalition of support across the community.

The bot would just tweet out the messages in a fixed interval in sequence until that campaign was completed / exhausted, and then would go dormant until another campaign was approved and launched.

Pull requests could be reviewed and discussed before going live, and perhaps even some type of governance system could be implemented for more fine grained consensus. 

This way we could target the promotion and education of the important features of Handshake, give balanced comparisons to other projects, like ENS, etc. (just rough ideas)

This way we can at least continue the conversation and not torpedo the potential reach of the account.

If it worked then it would be a major success for our community being aligned enough to collaborate in a selfless way and would demonstrate the emergence of decentralized governance for our community with respect to outreach.

Just a thought...

**Comment by realrasengan** [comment]
There seems to be so many differing opinions, and they all seem right when you look at it in each's context.  I think this will be difficult to come to a consensus if we approach it from a political manner.  So, instead, why don't we apply the same distribution model for names in Handshake to this twitter handle?  Let's decide this the same way the blockchain does.

Vickrey auction with proceeds going to one or all of the foundations?

**Comment by namervin** [comment]
FistFullofAss - there are programs like SocialPilot.co that my agency uses to give a set of people notifications of a new scheduled post submission - from there it can be edited, then someone has the final approval to send it out. I like it much better than hootsuite. As far as approval structure there maybe even better tools out there nowadays.

**Comment by Anunayj** [comment]
> handshake.org should not link nor endorse the @hns on Twitter and whoever controls it has the rights to publish his/her opinion as a Handshake director. See it as a perk for the early adoption.

Actually after reading [Falci's Response](https://github.com/handshake-org/handshake-web/issues/14#issuecomment-841681005). I'd change my view to just making it clear that the @hns handle is not the official one and removing it's links from handshake-web.

Making the Handle inactive seems like a awful waste of potential. And I feel like not having an official twitter handle won't negatively affect the coin at all. but it would make it clear to onlookers that the handle's view and biases does not reflect the community's.

It'd be nice if the twitter bio mentions that the twitter handle is not official. But that of course is the choice of it's owner.

**Comment by marcosmereles** [comment]
I'm probably a little late 😅 to the discussion about the fate of the @HNS Twitter account, but here's my take:

As HNS is a decentralization project, the internal organization should remain decentralized, democratic, seeking the participation of as many members of the community as possible, regardless of what they do within the community, as long as they are part of it. 🧐

As there are some developer members in the community, it is possible to create a web app pointed by a HNS domain.

An app where a vote is taken on which tweets to publish the next day, from a list of proposals where each member can vote only once for each tweet.

The app ID media can be HNS domains like Nomad.


To choose the options to tweet, it is necessary to establish some criteria, such as, for example:

-🚫HODL tweets
-🚫price tweets
-🚫other related projects references
-🚫Memes
-🚫any kind of promo or recommendation.
-✅ blockchain updates
-✅ recent HNS related projects
-✅ HNS related events
-✅ HNS new adoption browsers.


This is a rough idea, it is necessary to polish it in order to create something as close as possible to the essence of HNS ...

🤝DECENTRALIZATION🤝

**Comment by metanoia64** [comment]
@marcosmereles  did a nice job of breaking down the potential acceptance criteria of what to allow to tweet. RE: HNS related events, perhaps that could be kept to minimum as well.

**Comment by FistFullOfAss** [comment]
@marcosmereles the voting system that you're proposing for tweeting isn't very Sybil resistant. What's to stop me from getting a bunch of throw away domains off the secondary market for cheap and just voting with all of those skewing the vote in my favor. There is no real way for us to get actual community consensus on who runs the account or what types of post are on the account without someone exerting some kind of authority and putting someone in charge of the account. The account will always be seen as the official account for the HNS community even if rebranded as "not official", it's better to not leave the voice of the community into the hands of a few

**Comment by rafaelcastrocouto** [comment]
**Alternative Solution Number 2** (sorry I'm late)

Since someone from our community owns the @hns twitter handle, we could just add this description and pin a tweet declaring: 

"This account is managed by the following handshake community members: @X @Y and @Z"

This would make pretty clear that the account does not indicates any kind of centralization and would allow it to remain active (that seems kind of important to the project image).

About the link in the homepage, imho it would be better to add a few more successful accounts that have been mentioned here instead of removing it. This would make easier for beginners to join the community and would also diminish the centralization problem. 

About content, I also manage [a facebook page](https://www.facebook.com/handshake.org) and I think we all agree that those accounts (that might seem "official" for beginners) should avoid conflicts and try their best to represent the project with a respectful and professional behavior. How to guide those managers is a very important discussion but I'll leave it for another time.

**Comment by troq** [comment]
An alternative to keeping the twitter handle and homepage static would be to make both of them extremely permissive. ie @HNS could RT any project supporting Handshake with a budget of one tweet a week per project. RT namebase, hnsfund, dweb foundation, handy browser, shakedex, bob wallet, etc. Each project can tag @HNS to indicate which tweet they'd like to have RT'd that week. Tweets putting down other projects and misinformation should not be RT'd but anything positive can be fair game. 

The same can be done for the homepage. Instead of linking to only bob wallet and the handshake_hns telegram, handshake.org could link to all the projects listed above in random order (each project should submit a pr to be added). This would help Handshake come across as a vibrant, decentralized community (which it is) instead of a ghost town while also remaining neutral. 

I previously posted that I think @HNS should remain static, and I still think that's a good option, but this permissive approach may enable a better outcome for the community. First impressions matter and coming across as a dead project isn't ideal... Regardless, it's hard to fully predict what would be best without trying it. We could try the permissive approach first and time box it to 4 weeks, then pause and re-evaluate.

**Comment by agaamin** [comment]
@troq That is a great idea! :)
As an advertising and marketing person i totally agree with that statement "First impressions matter !"

**Comment by pinheadmz** [comment]
@troq I think thats clever - basically use @HNS as an amplifier for the diverse community already tweeting by themselves -- in short the rule would be that @HNS ONLY retweets ?

For handshake.org, I think that page should continue not to link to anything closed-source or for-profit but I agree with you that each add/remove link should be submitted as a PR and discussed. bitcoin.org has a [community page](https://bitcoin.org/en/community) where multiple forums are listed (of course they list /r/bitcoin but not /r/btc ...)

**Comment by Falci** [comment]
Could it be automated?

RT everything that mentions `@hns`, maybe with a minimum number o likes?
I know it opens room for spam, but I don't want someone moderating what should be tweeted and what should not.

**Comment by DIPMR** [comment]
Someone or any organization should not have the control& respect the genesis statements.

If I can not get access to the link as other got then this is a closed source project and only the foundation is director nobody else. There is no working around this problem.

Conflict: I still did not get the answer about the transfer of access.

**Comment by FistFullOfAss** [comment]
It seems like new opinions are slowing down now and the @hns Twitter is starting to retweet more with still zero transparency of who's running the account, so here's the vote for activating the twitter again:

For: 13
Chanjo
Pinheadmz
Jhonnywu
Alchmind
Peepo
Agaamin
Falci
hns-tld
NetOperatorWibby
robotsmadeit
Anunayj
marcosmereles
rafaelcastrocouto


Against: 9
Smcki
DIPMR
troq
Tynes
FistFullOfAss
Brandondees
Kiba
tiMaskal
chrxn

The conversation should be shifted to how the account is run now. I like what @pinheadmz was suggesting

> @Chjango opens a pull request to this repo that adds a new html page (i.e. handshake.org/twitter.html) that specifies the behavior of @hns with little room for interpretation. This will make the custodian's job easy, and provide continued influence by the community (who can open new PRs to update that policy).

This conversation should be mentioned somewhere in the new pinned tweet and it should be known that this conversation could be opened again if need be. Transparency and the ability for ownership to be transferred should be key for whoever admins the account.

**Comment by DIPMR** [comment]
These links are attached with the official repo of protocol and no matter what no body should have access to these no matter what the vote count is. This is ground 0.

**Comment by brandondees** [comment]
I don't think voting by way of github issue comment authorship is a valid method of community representation. Github issues is for exchanging ideas about pros and cons, not for consensus.

**Comment by DIPMR** [comment]
Do you guys think if founding members don't keep the genesis promises, then is anybody going to trust this product?

**Comment by FistFullOfAss** [comment]
@brandondees that's a fair point, but what should be used for community consensus. The @hns handle is becoming active again without consensus, rules, or transparency.

@DIPMR not sure what you mean by genesis promises and how it relates to the twitter handle becoming active again.

**Comment by brandondees** [comment]
> @brandondees that's a fair point, but what should be used for community consensus. The @hns handle is becoming active again without consensus, rules, or transparency.

Really not sure what would work well, or if consensus is even essential in this decision. We just need to persuade the account holders about what should be done with it, whether or not they want community vote to factor in is up to them. I'd suggest a polling website that can be shared across the many community channels rather than limiting to a platform

**Comment by smcki012** [comment]
The last confirmed owners were Caleb Chen and Andrew Lee of PIA VPN/London Trust Media. Caleb managed the Tweetdeck and had all the credentials in his name. If it has exchanged hands, it was done purposely. And likely this entire process is probably  a public charade to make it seem decentralized.

**Comment by Chjango** [comment]
> I agree with this generally but what sorts of community members do we want? Do retail investors add value to the community as a whole or are they there to attempt to extract value? (and then get dumped on by insiders)

For HNS, we don't have the luxury to choose imo. If we were in a bear market with a lot less competitors, it would be another story. (Have you seen the number of domain name projects popping up like mushrooms since the bull market started? They're much better funded and centrally operated and taking mindshare away from Handshake, despite being inferior products.) The other thing is, when HNS gets listed on major exchanges—which is the mecca that we're all aiming for—it will inevitably drive both retail and whales onto Handshake, and it would be out of anyone's control who does what with the coin.

That being said, every class of community member is valuable and has utility for the project; not just developers. Investors have their value because they lend much-needed liquidity to the coin as well as lend capital to bootstrap early projects building on top of Handshake (especially absent a centralized treasury). Retail has its value because they're the ones who either undervalue or overvalue a project and thus, when they do overvalue a coin, they pay a premium for it, which itself is the best form of "marketing" for a coin. If you look at the meteoric price rise of any DeFi coin, it's retail and investors who are the driving forces behind such price action (see: forward price-to-earnings on investopedia). Developers are certainly critical for the adoption of Handshake because they're the ones making everything fundamentally more valuable, assuming you're building something necessary which will make usage easier, but in the backdrop of this bull market, I think we need to be more pragmatic, less ideological.

**Comment by Chjango** [comment]
> Someone or any organization should not have the control& respect the genesis statements.
> 
> If I can not get access to the link as other got then this is a closed source project and only the foundation is director nobody else. There is no working around this problem.
> 
> Conflict: I still did not get the answer about the transfer of access.

I had been transferred access to tweet from the account. It started when a few people on twitter commented that the inactivity of the account is shedding a bad light on Handshake, which seemed like a problem with a straight-forward solution: start tweeting from the account and demonstrate how vibrant and active the community really is.

The framework I'd been using for tweeting is, based on the suggestions in this thread to:
- tweet only dev-related stuff

The framework I'd used for RT'ing is:
- extremely permissive RT'ing of all relevant (not spammy) mentions of HNS as well as testimonials from people who independently discovered Handshake based on the merits of the project

In general, I'm in agreement with @troq's suggestion to only RT (w/o comment) specific accounts no more than once a week. Those accounts can be listed on handshake.org, and anybody who wants "in" can open a PR to get it. This seems to be the most lightweight option that will allow all the directors (and future directors) on this thread to act like a DAO (without a blockchain or smart contract involved), to dictate what should be RT'd, and relegate the responsibility of the account's custodian to hitting the button to highlight the things we all want to highlight.

**Comment by tiMaxal** [comment]
not sure when i was made 'against' https://github.com/handshake-org/handshake-web/issues/14#issuecomment-846350246 - my only response has been rather 'on-the-fence' ..

> While only having started becoming part of this community a month or so ago, one of the things that appealed was the brevity and apparent non-politicisation of 'official' social media; keep it simple and keep it true, and if it _is_ going to kept going moving forward, keep it active within the month if not the week. Perspective of a recent 'outsider' :-).

**Comment by DIPMR** [comment]
> @brandondees that's a fair point, but what should be used for community consensus. The @hns handle is becoming active again without consensus, rules, or transparency.
> 
> @DIPMR not sure what you mean by genesis promises and how it relates to the twitter handle becoming active again.

Promise:   "True decentralization, no official singular Foundation, Committee, Corporation, or entities in permanent unitary control of the protocol." - handshake.org The links attached to the Protocol makes it a part of protocol and "1 person/entity" has control over it. I learned most of the tech from "learnhns.com' &  Boyma's presentation (https://www.youtube.com/watch?v=h505L7A_Z8g&t=9s), uploaded in March,2019 , even before the genesis but there is nothing mentioned about it. 

Don't know what kind of analysis to do here but  it's pretty clear and obvious that tweets are biased.

**Comment by DIPMR** [comment]
> I had been transferred access to tweet from the account. It started when a few people on twitter commented that the inactivity of the account is shedding a bad light on Handshake, which seemed like a problem with a straight-forward solution: start tweeting from the account and demonstrate how vibrant and active the community really is.


--This raises many questions in itself. 
Who (person/group/foundation/etc) transferred it? Can I  get the name and where was it discussed?  I see only 2 Tweets talking about HNS not tweeting or anything like that and there are not many responses to that tweet to call it a healthy discussion. Core of this project is to remove the MITM but I see the irony here by appointing a MITM at will and that too without even thinking about anything. If founders passed the right then I even question the founders and doubt who they are and why they are founders in first place. 

In the end whoever has the access has the power and he/she/they is MITM.

**Comment by kibagateaux** [comment]
> If we were in a bear market with a lot less competitors, it would be another story. (Have you seen the number of domain name projects popping up like mushrooms since the bull market started?

These aren't competitors. They aren't trying to replace the root zone they are making siloed namespaces, a lot of the time they aren't even related to DNS and just wallet utilities like ENS. Our only real "competitor" is ENS and even then they are a complimentary technology. Do you see them worrying about all the people forking their code? No, they just keep building because they know they are the best out there.

> when HNS gets listed on major exchanges—which is the mecca that we're all aiming for

No that is NOT the mecca. HNS is trying to replace the DNS root zone, not get listed on exchanges. You have failed to make and logical or convincing argument how shilling to retail investors helps us achieve Handshake's goal. Reiterating -> **Handshake's goals**, not your own goals.

>  If you look at the meteoric price rise of any DeFi coin, it's retail and investors who are the driving forces behind such price action 

DeFi is an entirely different asset class which is not comparable to HNS first off. Second, they have fundamental value backed by revenue generation, cash flows, and DAO treasuries none of which Handshake has. Third, DeFi adoption happened long before price appreciation. I've been heavy DeFi user since mid-2019 when no one was talking about it on twitter, all the tokens were sub $50M marketcap or didn't exist yet, etc. and I can tell you for a fact that price movement didn't do anything for their success, they were successful and so they became valuable. You don't even repost memes from HNS official twitter which is crypto marketing 101 so you're not even emulating the most basic tactics used by these communities, you're just pushing your own narrative.

**Comment by agaamin** [comment]
Lol. I think before we decide what to do with the social media assets of
handshake, we should seek to arrive at a process/system on how we can
arrive at a consensus on various topics in the HNS community.

In the long run there will be all kinds of people in a community such as
Handshake. There will be those who want to contribute to and evangelize
open source tech like Handshake and those who want to create innovative
products, services and businesses out of them and even some who may just
want to buy and sell TLDs or Tokens to make money.

All of them are perfectly acceptable manifestations of a movement. In a
diverse community it would be great if we can develop a mechanism of
arriving at a consensus on various issues.

What would that consensus look like ? Flat Voting systems or Weighted
voting systems or some other method maybe. How can such a consensus be
implemented?

It might help us manage and accommodate different views and still keep
moving forward. Decisions on what to do with various assets and aspects of
HNS can be made after that so that there are no heartburns in hindsight.

Cheers
Sajan

**Comment by DIPMR** [comment]
> > I had been transferred access to tweet from the account. It started when a few people on twitter commented that the inactivity of the account is shedding a bad light on Handshake, which seemed like a problem with a straight-forward solution: start tweeting from the account and demonstrate how vibrant and active the community really is.
> 
> --This raises many questions in itself.
> Who (person/group/foundation/etc) transferred it? Can I get the name and where was it discussed? I see only 2 Tweets talking about HNS not tweeting or anything like that and there are not many responses to that tweet to call it a healthy discussion. Core of this project is to remove the MITM but I see the irony here by appointing a MITM at will and that too without even thinking about anything. If founders passed the right then I even question the founders and doubt who they are and why they are founders in first place.
> 
> In the end whoever has the access has the power and he/she/they is MITM.

----------Still waiting on the details.  It's been a while and a response will help to clear the blockage to understand this ecosystem and foundation better to make better decisions. @Chjango

**Comment by smcki012** [comment]
There is always the nuclear option of just getting the account suspended permanently and moving on. There is no need for a Twitter account to rally a community -- we already have one. Twitter is tiny compared to other sites like TikTok, etc. 

Instead of continuing this charade where the DWeb Foundation members try to claim social legitimacy, I think it's best they stop this now before it devalues their non-profit efforts to build the community before theirs even exists. 

This isn't Tezos or Ethereum -- and the less voices arguing and trying to centralize clout, the faster HNS grows. If those in control of the account can't see that, they shouldn't have been given access by Caleb Chen of London Trust Media anyways. 

We all agreed on a document (Andrew Lee, Andrew Lee, Caleb Chen, Zipkin, myself and others) that the account would be *permanently* forfeited to prevent Namebase or anyone else from centralizing control. 

Be mindful of the replies in this thread and people's ulterior motives. They just want something to fight for. But the more this goes on, the less the account even matters.

**Comment by Handshake0007** [comment]
I’m against any centralized control or controllers. Just like the founders killed the original Foundation, we should stay the course, and execute @HNS twitter. I vote for a Gravestone... communicating that Handshake is a leaderless, decentralized, utility for public good.

**Comment by alchmind** [comment]
Interesting discussion about centralization or centralized controllers - so should anyone who has a significant following on Twitter posting about HNS be asked to shut down their accounts or be regulated in what they say as well?

If we just shut down the @HNS account then others posting about @HNS would have much more clout with less twitter competition.

**Comment by smcki012** [comment]
> If we just shut down the @HNS account then others posting about @HNS would have much more clout with less twitter competition.

Yes, that is the very essence of decentralization. There is no brand or entity that is *formally* "HNS". Only for-profit, non-profit, or community contributors with their own prerogatives and communities. That is **emergent governance**.

**Comment by skyinclude** [comment]
I saw a lot of "competing namespaces" (i know people say that they are not competitors - but I would say any namespace taking mindshare away from hns is a competitor - direct or indirect) use the inactive HNS twitter as a reason to call HNS dead.

My vote is to make it a bot / formula (if there is a way) that can be done to share all the amazing things in the community so it doesnt turn into a spam machine. maybe with the $hns tag. If there is a tool that allows the community to vote and what is approved is posted to the official HNS.

HNS is an amazing project - but without marketing or even using a marketing asset is built up - and linked on official websites and exchanges - seems like a waste of potential to get more people to know Handshake.

At the very least a pinned message at the top of the account explaining why it isn't active - very clearly.

"This Twitter account is not active, as like bitcoin, there is no centralized authority"

**Comment by johnnywu-namebase** [comment]
I'm against using a bot since it'd be really easy to hijack the content, and I'm very much against deleting the @HNS account outright. While I can get behind freezing it and pinning a Tweet on @HNS that shares why the account is not active and to visit $HNS and #HNS for the latest news, I don't think this really makes sense either because then what about the Handshake LinkedIn group, the Handshake Facebook page, the Handshake Instagram page, the Handshake subreddit, the Telegram group, etc? To freeze just the Twitter account and not any of the other platforms that are also ran by individual community members seems kinda... weird?

I'm leaning heavily toward's @troq's suggestion:

> An alternative to keeping the twitter handle and homepage static would be to make both of them extremely permissive. ie @hns could RT any project supporting Handshake with a budget of one tweet a week per project. RT namebase, hnsfund, dweb foundation, handy browser, shakedex, bob wallet, etc. Each project can tag @hns to indicate which tweet they'd like to have RT'd that week. Tweets putting down other projects and misinformation should not be RT'd but anything positive can be fair game. 

In addition, I think @HNS should be regularly RTing positive Handshake memes since they can be super helpful in educating and oboarding newcomers. Our community seems to regularly look to Bitcoin for inspiration, and their Twitter account also posts memes (see: https://twitter.com/Bitcoin).

> The same can be done for the homepage. Instead of linking to only bob wallet and the handshake_hns telegram, handshake.org could link to all the projects listed above in random order (each project should submit a pr to be added). This would help Handshake come across as a vibrant, decentralized community (which it is) instead of a ghost town while also remaining neutral.

Other decentralized projects' community pages already do something similar to this (see: https://www.arweave.org/get-involved/community and https://bitcoin.org/en/community).

**Comment by nlydv** [comment]
Having just now seen this discussion for the first time, and neither being entirely clued into the specifics of the account's origin or developments nor giving the time to read every comment here word for word, here are my thoughts:

It appears there are two main categories of solutions proposed:

* Kill it
* Mimic

Either we kill it, and whatever perceived future risk the @hns handle holds, or we try to structure the account in a way that mimics Handshake's (or blockchain's general) modes of decentralization at various levels (code, networking, community, etc.).

The latter is difficult because Twitter. The former is undesirable because... Twitter.

If we're trying to optimize for decentralization, weighing the socio-political parameters of one specific facet of Handshake's public face on its own, its pseudo-official Twitter account, is futile. Governance in the abstract when aiming for democratic/decentralized principles (at least in this context) should be cohesive. 

Obviously others have also discussed the related topics of the subreddit, other channels, and the public links to and from them, but the Twitter is special because it undoubtedly has the highest potential for public outreach/discovery.

What about GitHub? Is the @handshake-org account—being a centralized host of the website, the actual code, and the only thing that's agreeably, officially Handshake—supposed to be a standard for other public channels? Does putting that level of trust in Microsoft, however far removed and mitigated by git and open collaboration, draw a line on what is acceptably decentralized while balancing convenience and practicality (i.e. is this what we theoretically want to mimic)?

I'm not asking all that to be critical of the use of GitHub at all. Just positing questions in an armchair philosopher kind of way. The point though is that, since there is an actual substantive debate over the Twitter account, it's probably better to consider/develop Handshake's entire public-facing structure, community governance (or lack thereof), social consensus mechanisms, etc. as a whole and then apply it to specific facets.

I don't have specific ideas for who, what, when, where, or how; just the why. Perhaps some non-permanent federated-style of  independent working groups, maybe just a non-binding document outlining desired social practices for community members. Hell, I don't know.

But I can't see how we can efficiently progress Handshake without some overarching... principles? organization? guidelines? ... some overarching social structure, in some form... at least in the short-term. To do nothing and/or leave it to the incoming public (whether individuals, organizations, companies) to vie for influence over the public face of Handshake will not be good in the long-term.

This Twitter thing, the struggle to figure out Brave integration, other misc. interactions with other communities, and the ever-looming possibility for a sudden influx of unsavory/bad actors co-opting the technology to propagate extreme ideologies, general ill will, or even illegal activities, in the process cratering Handshake's reputation. Adopting "radical decentralization" can only raise these barriers to progress.

If it's not clear, **_I'm opposed to killing the Twitter account_**. That is a rash decision to make without a more holistic consensus on how to guide Handshake's future on the social layer—layer 2, if you will—after which, if it's decided that it must die, then we can write a eulogy, pin it to the top of its timeline, and send it down the river styx. 

...

ok, this meandering unloading of thoughts has concluded.

**Comment by pinheadmz** [comment]
https://twitter.com/HNS/status/1426647182581579783

Should we close this issue now?

![Screenshot from 2021-08-14 17-37-06](https://user-images.githubusercontent.com/2084648/129460647-9fda9f05-b6f1-4e91-b2c4-edfab59df4c8.png)
![Screenshot from 2021-08-14 17-36-57](https://user-images.githubusercontent.com/2084648/129460648-f273e7bd-cda8-474c-9b1b-89e01d6c7c2d.png)

**Comment by DIPMR** [comment]
I don't think we found any kind of consensus on this issue.

**Comment by smcki012** [comment]
Yes we have. It's clear HNS doesn't need the Twitter account to succeed or get adoption (i.e see the community efforts for the Namecheap integration). 

Any further pushing on this matter from new community contributors/members with no context on the chain's history  should be seen as an attempt at social governance capture. 

We're not stupid.

**Comment by DIPMR** [comment]
Everyone/anyone can think whatever they can about anything. This thread already contains ugly facts of ownership transfer.
If we don't want to do anything about this then the best thing is to delete this thread. Not commenting or not doing anything is not gonna change anything about it.

**Comment by skyinclude** [comment]
Yes, lets close this case - but someone on Twitter commented they could make a Bot 

https://twitter.com/AGreenDCBike/status/1426889381634363398

not sure if they are saying they can make it - or suggest someone else make it?
